src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -51,7 +51,7 @@ export default () => `
 					</div>
 					<div class="row header-bottom-nav-tiles ads-grid">
 						<div class="col">
-							<a class="header-bottom-nav-tile-link" href="command:registeredServers.addConnection">
+							<a class="header-bottom-nav-tile-link" href="command:registeredServers.addConnection" role="button" name="Create a Connection">
```

Comment:
is there any reason you are using name attribute? I think you should use aria-label here, also the text should be localized. this comment also applies to other changes in this PR.

Reply:
@alanrenmsft I was only doing this because of this comment left on the issue..

"The new control should be set as button with **name** as "New""

Reply:
I am pretty sure they don't actually mean html name attribute. please use aria-label

Reply:
@alanrenmsft switched to aria-label